src/algorithms/linear_stretching.jl

```diff
@@ -2,7 +2,7 @@
 """
 ```
     LinearStretching <: AbstractHistogramAdjustmentAlgorithm
-    LinearStretching(; minval = 0, maxval = 1)
+    LinearStretching(; minval = 0, maxval = 1, [src_minval], [src_maxval])
```

Comment:
for symmetry, maybe `minval` and `maxval` should be `dst_minval` and `dst_maxval` ?

Reply:
I've thought about that once a while, I think it's okay to keep the previous `minval, maxval` name since those are what most user needs.

And I guess adding another deprecation after `imadjustintensity ` is kind of annoying to users...

Reply:
i'm not sure about "most users needs".  i for one would only ever need to set `src_{min,max}val`.  worth noting that matlab's [equivalent function](https://www.mathworks.com/help/images/ref/imadjust.html) uses symmetrically named keyword args (`{low,high}_{in,out}`).  i would also argue that naming them `dst_{min,max}val` would be more self-descriptive and hence act as documentation, both in the user-facing API and in the code.

Reply:
Yes, I agree and the re-naming makes sense to me as well. I was just concerning how welcome/unwelcome this renaming will be.

I'm open to changes, but I'd like to first see how @zygmuntszpak thinks before I put my hands on it.

Reply:
I agree that it is best to rename it to `dst_minval` and `dst_maxval`. Presumably, this means that we will have to amend the version number to become `0.4.0` to reflect the breaking change?

Reply:
> Presumably, this means that we will have to amend the version number to become 0.4.0 to reflect the breaking change?

If we're on `1.x.y` version, a minor version bump is needed. But since we're on `0.3.4` version, _I think_ a patch version is also accepted.

> When you deprecate part of your public API, you should do two things: (1) update your documentation to let users know about the change, (2) issue a new minor release with the deprecation in place
> https://semver.org/#how-should-i-handle-deprecating-functionality

Reply:
In addition to the deprecation of `minval, maxval` in favor of `dest_minval, dest_maxval`, I also introduced a new set of API in 5674568

* specify (dest_minval, dest_maxval): `LinearStretching(nothing=>(0, 1))`
* specify (src_minval, src_maxval): `LinearStretching((0.1, 0.9))`
* specify both: `LinearStretching((0.1, 0.9) => (0, 1))`